### ISSUE TITLE
miniconda not working right

### ISSUE THREAD

**Issue by PatrickPaul-NRC** [ai4d-iasc/trixie#14]
Name of the module: miniconda3-4.7.12.1-gcc-9.2.0-j2idqxp

Steps to reproduce:
```bash
module load miniconda3-4.7.12.1-gcc-9.2.0-j2idqxp
conda create -c conda-forge -p ~/test/miniconda_fail python=3
```
:ok_hand: - Up until here 

:warning: The following fails:
```bash
conda activate ~/test/miniconda_fail
```

Trying to run ```conda init bash``` tries to invoke conda using sudo ??? and of course this is not what we want.

BTW, there is a newer version of miniconda (latest version: 4.8.3) at the time of writing this issue

Please see transcript of this:
[reproduce.txt](https://github.com/ai4d-iasc/trixie/files/4827018/reproduce.txt)

**Comment by joeydumont** [comment]
Could you try with `source activate ~/test/miniconda_fail`? It seems that `conda init` messes with the installation files (that are not world-writable) and breaks conda. This is how the workflow is documented in the [wiki](https://github.com/ai4d-iasc/trixie/wiki/jobs-conda-jupyterlab).

I've installed miniconda3 4.8.2, accessible with `module load miniconda3-4.8.2-gcc-9.2.0-sbqd2xu`. You'll encounter the same issue though.

Note that the software stack on trixie is under heavy development, so expect things to change soon. Also note that Compute Canada [discourages the use of Anaconda and conda](https://docs.computecanada.ca/wiki/Anaconda/en) on their clusters.

**Comment by PatrickPaul-NRC** [comment]
Yes I found out about source activate and it works.

As long as there is a mean to get to where I need I don’t care too much how I get there.

I am not aware of compute Canada, never used it. But one thing that is super important for me and my team is reproducibility. So it doesn’t help me very much if the method to have access to a software is only valid on Trixie. When I communicate with my clients I need to have an easy way to convey how to reproduce the results I am presenting to them. This is critical in my business.

Miniconda seems to have gotten me where I needed most of the time. 

Of course we don’t have all the same prerogatives and there is definitely huge gains from having a set of tools available centrally. 

One thing I learned over the years especially in supporting HPC communities, is that it’s never a one size fits all solution. So if it doesn’t hurt anyone or break anything, let people experiment you might learn a few things along the way.

Thanks for the info and the newer conda I will try that out
